.github/workflows/bindings.yml

```diff
@@ -29,6 +29,11 @@ jobs:
     needs: build-ios
     runs-on: macos-latest
```

Comment:
Unrelated to this PR, but is there a reason why we're not using our runner for `test-ios`?